cpu/cc26x0/include/cc26x0_prcm.h

```diff
@@ -296,13 +296,19 @@ typedef struct {
 #define PDSTAT0_SERIAL_ON    0x2
 #define PDSTAT0_PERIPH_ON    0x4
 
+#define PDSTAT0RFC_RFC_ON    0x1
+
 #define PDCTL1_CPU_ON       0x2
 #define PDCTL1_RFC_ON       0x4
 #define PDCTL1_VIMS_ON      0x8
 
 #define PDSTAT1_CPU_ON      0x2
 #define PDSTAT1_RFC_ON      0x4
 #define PDSTAT1_VIMS_ON     0x8
+
+#define PDSTAT1RFC_RFC_ON    0x1
+
+
```

Comment:
could these go into a separate commit?
(preferably, one we're they're also being used  - like "cpu/cc2650: wait for the RFC to power on")